java-frontend/src/main/java/org/sonar/java/bytecode/se/BytecodeEGWalker.java

```diff
@@ -858,66 +858,76 @@ private boolean isExceptionHandledByBlock(Type exceptionType, BytecodeCFG.Block
   void handleBlockExit(ProgramPoint programPosition) {
     BytecodeCFG.Block block = (BytecodeCFG.Block) programPosition.block;
     Instruction terminator = block.terminator();
+    if (terminator == null) {
+      enqueueHappyPath(programPosition);
+      return;
+    }
+    switch (terminator.opcode) {
+      case GOTO:
+        enqueueHappyPath(programPosition);
+        break;
+      case TABLESWITCH:
+      case LOOKUPSWITCH:
+        programState = programState.unstackValue(1).state;
+        enqueueHappyPath(programPosition);
+        break;
+      default:
+        handleBranching(terminator);
+    }
+  }
+
+  private void handleBranching(Instruction branchInstruction) {
```

Comment:
To keep vocabulary small and consistent why not rename this parameter terminator  ?